### PR TITLE
Fix swagger not handeling server errs

### PR DESCRIPTION
**Pull request by Audakel**

### CODE REVIEW

**Comment by kupiakos** [comment]
You didn't pull in the changes I made to the SwaggerHandler -_-

**Comment by Audakel** [comment]
what changes? I rewrote most of the SwaggerHanlder

**Comment by kupiakos** [comment]
Look at the code that's being removed in `SwaggerHandler`. That stream handling code is much simpler, and faster. Also, `private SwaggerHandler swaggerHandler = new SwaggerHandler();` is completely unneeded and should be removed. Did you pull in the changes before working on this?

File: java/Phase 1.iml [hunk 1]
```diff
@@ -28,5 +28,7 @@
     <orderEntry type="library" name="byte-buddy-1.4.33" level="project" />
     <orderEntry type="library" name="objenesis-2.4" level="project" />
     <orderEntry type="library" name="annotations-13.0" level="project" />
+    <orderEntry type="library" name="junit-4.11 (3)" level="project" />
+    <orderEntry type="library" name="hamcrest-core-1.3 (3)" level="project" />
```
Comment (kupiakos) [inline java/Phase 1.iml:32]: Where is hamcrest being used?

File: .idea/inspectionProfiles/Project_Default.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <component name="InspectionProjectProfileManager">
-  <profile version="1.0">
+  <profile version="1.0" is_locked="false">
```
Comment (kupiakos) [inline .idea/inspectionProfiles/Project_Default.xml:2]: What are these for?

File: .gitignore [hunk 3]
```diff
@@ -60,3 +60,5 @@ java/build/
 java/module_phase_1.xml
 java/testreport/
 testreport/
+.idea/
+java/Phase 1.iml
```
Comment (kupiakos) [inline .gitignore:64]: Why? These two shouldn't need to be here. `Phase 1.iml` should be updated when we add components, etc.

File: java/Phase 1.iml [hunk 4]
```diff
@@ -28,5 +28,7 @@
     <orderEntry type="library" name="byte-buddy-1.4.33" level="project" />
     <orderEntry type="library" name="objenesis-2.4" level="project" />
     <orderEntry type="library" name="annotations-13.0" level="project" />
+    <orderEntry type="library" name="junit-4.11 (3)" level="project" />
```
Comment (kupiakos) [inline java/Phase 1.iml:31]: What is this extra JUnit?

**Pull request closed without merging**